model/User.js

```diff
@@ -0,0 +1,66 @@
+'use stirct';
```

Comment:
*TODO FOR LOST POINTS*

What is wrong with this at line 1? This breaks your tests.